api-extractor/src/Extractor.ts

```diff
@@ -1,3 +1,5 @@
+/* tslint:disable:no-constant-condition */
```

Comment:
General note: The prototype was forgetting to collect references for these things:
- property getters/setters
- type literals
- implemented interfaces for a class (although we do follow base classes)
- (?) not sure about base interfaces for an interface
- function parameters